common/djangoapps/track/backends/mongodb.py

```diff
@@ -46,6 +47,8 @@ def __init__(self, **kwargs):
         db_name = kwargs.get('database', 'track')
         collection_name = kwargs.get('collection', 'events')
 
+        auth_source = kwargs.get('auth_source', '') or None
```

Comment:
@xitij2000, wouldn't the below code do the same since the `.get()` method returns `None` if the key is not found?
```suggestion
        auth_source = kwargs.get('auth_source')
```